### ISSUE TITLE
Modify the loss function

### ISSUE THREAD

**Issue by wysot** [abnerwang/py-Vital#4]
Excuse me, if I want to modify the loss function, should the loss function of the training phase and the loss function of the tracking phase be changed? Or just change the loss function of the training phase?
Thank you

**Comment by abnerwang** [comment]
I think all the loss functions should be changed.

> Excuse me, if I want to modify the loss function, should the loss function of the training phase and the loss function of the tracking phase be changed? Or just change the loss function of the training phase?
> Thank you

I think all the loss functions should be changed.

**Comment by wysot** [comment]
Thank you very much.

**Comment by wysot** [comment]
Thank you very much for your work.
the result on OTB2015 is 0.6708, but the result on the paper is 0.68. Can your result reach 0.68?
Thank you in advance.

**Comment by abnerwang** [comment]
> Thank you very much for your work.
> the result on OTB2015 is 0.6708, but the result on the paper is 0.68. Can your result reach 0.68?
> Thank you in advance.

You need to adjust the hyperparameters according to your needs.

**Comment by wysot** [comment]
The article has repeatedly mentioned that they only use G in the training phase, and only use D in the testing phase. However, the code did not find the use of G during the training phase. Instead, it used G in the test phase. How do you correspond to this part?

**Comment by abnerwang** [comment]
> The article has repeatedly mentioned that they only use G in the training phase, and only use D in the testing phase. However, the code did not find the use of G during the training phase. Instead, it used G in the test phase. How do you correspond to this part?

You didn't understand the paper and didn't understand the code. You can read it carefully several times.
